src/main/java/core/basesyntax/service/reader/ReaderServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.reader;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> readFromFile(String inputDataFile) {
+        Path filePath = Paths.get(inputDataFile);
+        List<String> data = Collections.emptyList();
+        try {
+            data = Files.lines(filePath)
```

Comment:
return Files.lines(Path.of(inputDataFile))
and you can delete return data because you will throw an exception if you will get exception here